app/src/main/java/com/github/multimatum_team/multimatum/service/ProcrastinationDetectorService.kt

```diff
@@ -34,23 +32,25 @@ class ProcrastinationDetectorService : Service(), SensorEventListener {
     private var isServiceStarted = false
     private var wakeLock: PowerManager.WakeLock? = null
 
-    // data relative to the last time a movement was detected
-    private var lastDetectionTimestampNanos: Long = LAST_DETECTION_TIMESTAMP_NONINIT_CODE
-    private var lastPosition: Array<Float>? = null  // null only at initialization
+    private val binder = PdsBinder()
 
-    // the service will wait for this number of detections before starting to display toasts
-    private var nonReportedDetectionsCnt = NON_REPORTED_DETECTIONS_CNT_INIT
+    private lateinit var sensorListener: ProcrastinationDetectorSensorListener
 
-    private val binder = PdsBinder()
+    override fun onAccuracyChanged(sensor: Sensor?, accuracy: Int) =
```

Comment:
Could it be possible to make a test for this function too ? (Just to call it to increase coverage based on codeclimate)